tests/fixtures/entity-1/entity.pem

```diff
@@ -0,0 +1,6 @@
+This private key is only used for unit tests, please do not report it as some
```

Comment:
> invalid PEM file permissions 644

Will need to fix the permissions before running tests as git only preserves the execute bit :grin: